Public/Src/Deployment/buildXL.dsc

```diff
@@ -21,9 +21,9 @@ namespace BuildXL {
             // analyzers
             importFrom("BuildXL.Tools").Execution.Analyzer.exe,
             importFrom("BuildXL.Tools").BxlScriptAnalyzer.exe,
-            ...(qualifier.targetFramework === "netcoreapp2.2" ? [] : [
+            ...addIf(qualifier.targetFramework !== "netcoreapp2.2",

```

Comment:
>ifier.targetFramework !== "netcoreapp2.2 [](start = 25, length = 40)

If we will never support this on .net core (i.e. not even on 3.0 we should use `addIf(BuildXLSdk.isFullFramework,`

Reply:
Speaking of, I don't know a single reason why this should not run on macOS?

Reply:
Adding it.